actors/paych/src/lib.rs

```diff
@@ -42,17 +43,25 @@ pub const ERR_CHANNEL_STATE_UPDATE_AFTER_SETTLED: ExitCode = ExitCode::new(32);
 
 /// Payment Channel actor
 pub struct Actor;
+
 impl Actor {
     /// Constructor for Payment channel actor
     pub fn constructor(rt: &mut impl Runtime, params: ConstructorParams) -> Result<(), ActorError> {
         // Only InitActor can create a payment channel actor. It creates the actor on
         // behalf of the payer/payee.
         rt.validate_immediate_caller_type(std::iter::once(&Type::Init))?;
 
-        // Check both parties are capable of signing vouchers
-        let to = Self::resolve_account(rt, &params.to)?;
+        // Resolve both parties if necessary.
+        // Note that this does NOT guarantee that the parties exist in state yet.
+        let to =
+            rt.resolve_address(&params.to).with_context_code(ExitCode::USR_NOT_FOUND, || {
```

Comment:
The prior code returned `USR_ILLEGAL_ARGUMENT` here (in `resolve_to_actor_id` as well as if the actor type was wrong). The FRC46 token uses `USR_NOT_FOUND`.

In the first instance, I think it's better not to change behaviour right now, so use ILLEGAL_ARGUMENT. However, this address resolution thing strikes me as fundamental enough that maybe we should dedicate a new user exit code for it? Let's discuss in Slack.